lib/dynamodb-integration.js

```diff
@@ -72,6 +78,27 @@ const buildPayload = (tableName, streamName, events, additionalAttributesSchema
     };
   };
 
+const isDynamoDbError = (err, data) => err || !isEmpty(data.UnprocessedItems);
+
+const sendLogsToDynamoDb = (dynamodbClient, payload, cb, times = 3) => new Promise((resolve) => {
+  dynamodbClient.send(new BatchWriteItemCommand(payload), function (err, data) {
+    debug('sent to aws, err: ', err, ' data: ', data)
+    if (isDynamoDbError(err, data) && times > 0) {
+      debug('error during batchWriteItem', err, true)
+
+      if (!isEmpty(data.UnprocessedItems)) {
```

Comment:
extract the in condition into a func and use it in every place (here and `isDynamoDbError`)

Reply:
38f44358b6c1ddbf27844bfa870841cf278509aa